packages/calcite-components/src/components/table/table.tsx

```diff
@@ -0,0 +1,524 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Listen,
+  Prop,
+  State,
+  VNode,
+  Watch,
+} from "@stencil/core";
+import { Scale, SelectionMode } from "../interfaces";
+import {
+  LoadableComponent,
+  setComponentLoaded,
+  setUpLoadableComponent,
+} from "../../utils/loadable";
+import {
+  connectMessages,
+  disconnectMessages,
+  setUpMessages,
+  T9nComponent,
+  updateMessages,
+} from "../../utils/t9n";
+import {
+  connectLocalized,
+  disconnectLocalized,
+  LocalizedComponent,
+  numberStringFormatter,
+  NumberingSystem,
+} from "../../utils/locale";
+import { TableLayout, TableRowFocusEvent } from "./interfaces";
+import { CSS, SLOTS } from "./resources";
+import { TableMessages } from "./assets/table/t9n";
+import { getUserAgentString } from "../../utils/browser";
+
+/**
+ * @slot - A slot for adding `calcite-table-row` or nested `calcite-table` elements. Content placed here will be rendered in a `table-body` tag.
+ * @slot table-head- A slot for adding `calcite-table-row` and nested `calcite-table-header` elements.
+ * @slot table-foot- A slot for adding `calcite-table-row` and nested `calcite-table-header` elements.
+ * @slot selection-actions - A slot for adding a `calcite-action` or other element to display when `selectionMode !== "none"` and `calcite-table-row` are selected.
+ */
+
+@Component({
+  tag: "calcite-table",
+  styleUrl: "table.scss",
+  shadow: true,
+  assetsDirs: ["assets"],
+})
+export class Table implements LocalizedComponent, LoadableComponent, T9nComponent {
+  //--------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  //--------------------------------------------------------------------------
+
+  /** When `true`, displays borders in the table */
+  @Prop({ reflect: true }) bordered = false;
+
+  /** Specifies an accessible title for the component. */
+  @Prop() caption!: string;
+
+  /** When `true`, number values are displayed with a group separator corresponding to the language and country format. */
+  @Prop({ reflect: true }) groupSeparator = false;
+
+  /** Specifies the layout of the component. */
+  @Prop({ reflect: true }) layout: TableLayout = "auto";
+
+  /** When `true`, displays the position of the row in numeric form */
+  @Prop({ reflect: true }) numbered = false;
+
+  /** Specifies the Unicode numeral system used by the component for localization. */
+  @Prop({ reflect: true }) numberingSystem?: NumberingSystem;
+
+  /** Specifies the page size of the component. When true, renders pagination. */
```

Comment:
Add backticks to true and specify the calcite reference to pagination:

```
Specifies the page size of the component. When `true`, renders `calcite-pagination`.
```

Reply:
Cool, in the future, if the provided UI updates to include other elements for page length, go to page, etc., we can update this, but for now its accurate to say just `calcite-pagination`.